core/protocols/aws-xml-protocol/src/main/java/software/amazon/awssdk/protocols/xml/AwsXmlProtocolFactory.java

```diff
@@ -100,10 +107,32 @@
             staxOperationMetadata.isHasStreamingSuccessResponse());
     }
 
+    protected <T extends AwsResponse> Function<AwsXmlUnmarshallingContext, T> createResponseTransformer(
+        Supplier<SdkPojo> pojoSupplier) {
+
+        return new AwsXmlResponseTransformer<>(
+            XmlProtocolUnmarshaller.builder().build(), r -> pojoSupplier.get());
```

Comment:
Add a create() to XmlProtocolUnmarshaller?

Reply:
Will fix.